lib/numbers_and_words/strategies/figures_converter/languages/nl.rb

```diff
@@ -24,6 +24,19 @@ def simple_number_to_words
               super
             end
           end
+
+          def number_without_capacity_to_words
+            if figures.number_in_capacity(1) == 1
+              if figures.hundreds
+                teen_hundreds = @figures[2, 2].to_figures
+                [([@translations.teens(teen_hundreds), translate(:hundreds, 1)] + simple_number_to_words).join]
+              else
+                simple_number_to_words + [translate(:mega, 1)]
+              end
+            else
+              super
+            end
```

Comment:
This is just a personal preference, how about?:

```ruby
return super unless figures.number_in_capacity(1) == 1

if figures.hundreds
  teen_hundreds = @figures[2, 2].to_figures
  [([@translations.teens(teen_hundreds), translate(:hundreds, 1)] +
    simple_number_to_words).join]
else
  simple_number_to_words + [translate(:mega, 1)]
end
```

Reply:
Yes, that's better. Updated.